src/guide/faqs.md

```diff
@@ -0,0 +1,100 @@
+### Can I run my code on GPU-accelerated hardware?
+
+Yes you can! GPU support is available and has to be enabled for your account during our **Private Beta**. Read [this section](/guide/configuring-labs.html#Hardware-configuration) of our documentation to learn how to configure it. We also have plenty [GPU related questions](#gpu)!
+
+### How do I get started?
+
+We know that getting started with Machine Learning is hard. That's why we've created a [Quickstart guide](/guide/quickstart.html) that should get you up and running in no time! It's also always good to keep an eye on our [examples](/examples/index.html).
+
+### How do I become part of the Private Beta?
+
+Easy. Simply visit [our app](https://machinelabs.ai) and login with your GitHub account. We onboard new users weekly.
+
+### How do I get access to datasets?
+
+You have unlimited internet access in your lab, which means you can access any dataset you like. The easiest way to fetch datasets would be to specify them as inputs in the ml.yaml file. Checkout our docs on [Lab Inputs](/guide/configuring-labs.html#Lab-inputs-datasets) to learn how to do that.
+
+We're also working on a new feature that lets you mount any data of any size at runtime of your lab, so you don't have to download your inputs every time you hit the run button.
+
+### How can I save and download output data?
+
+All you have to do is to write your generated files to `./outputs` and we'll make sure to upload them once your execution is done, so they become available in the outputs view. Take a look at [this lab](https://machinelabs.ai/editor/rJQrQ5wjZ/1506415557004-HkTTQ5Dob?file=ml.yaml&tab=outputs) to see it in action.
+
+Another way to access any generated data is via our [REST API](https://blog.machinelabs.ai/2017/10/16/new-rest-api-and-folder-support/).
+
+### How can I access trained models from other labs?
+
+This can be easily done by combining [Lab inputs](/guide/configuring-labs.html#Lab-inputs-datasets) with our [REST API](https://blog.machinelabs.ai/2017/10/16/new-rest-api-and-folder-support/)
+
+## GPU
+
+### How do I enable GPU suppor for my account?
+
+There's two ways to enable GPU support for you account **during our private beta**:
+
+- [Become a Patreon](https://www.patreon.com/machinelabs) for the MachineLabs project
+- Create awesome labs that others can learn from
+
+If you think the latter applies, please reach out to us [via email](mailto:hello@machinelabs.ai) so we can enable GPU support for you right away!
+
+### Why do I have to pay for GPU support?
+
+GPU is very expensive. We are a small team with a tight budget trying to move the whole Machine Learning community forward. We believe it is important to not leave AI exclusively to the bigger companies but rather have some small indie teams go working on it as well. You can make a difference by supporting us on our mission!
+
+### How much do I need to pledge?
+
+We  enable GPU support for all backers that are backing $10 or more per month. 
```

Comment:
Is there an extra space between _We_ and _enable_?

Reply:
yupp, fixed